src/demo/player.rs

```diff
@@ -63,7 +65,7 @@ fn spawn_player(
         Name::new("Player"),
         Player,
         SpriteBundle {
-            texture: image_handles[ImageHandles::PATH_DUCKY].clone_weak(),
+            texture: player_assets.ducky.clone(),
```

Comment:
@alice-i-cecile I remember you explicitly wanted a `clone_weak` here. I personally would prefer the `clone` shown here, but just wanted to let you know

Reply:
In theory, this is fine because we're holding the handle elsewhere. In practice, I don't want to encourage users to clone_weak() in these cases because we really do want to tie the lifetime of the audio asset to the clips playing it. The cost of incrementing an Arc is chump-change in this context. Users that are aware of the tradeoffs here are welcome to optimize, but this is aggressively premature optimization _in the context of educating new bevy users_.

Reply:
Fully agree with that assessment.